exercise/templatetags/exercise.py

```diff
@@ -117,81 +117,68 @@ def percent(decimal):
 def submission_status(status):
     return Submission.STATUS[status]
 
-
+def _get_points_type(context):
+    points_type = ""
+
+    if ((context['confirm_the_level']) or
+        context['difficulty'] == "-" and context['max'] <= 2):
+
+        if (context['full_score'] or
+            (context['passed'] and context['confirm_the_level'])):
+            points_type = "accepted"
+        elif context['passed'] and not context['confirm_the_level']:
+            points_type = "ok"
+        elif ((context['points'] == 0 and context['submitted'])
+            or context['graded']):
+            points_type = "rejected"
+        elif context.get('confirmable_points'):
+            points_type = "missing"
+        else:
+            points_type = "not_submitted"
+    else: # normal exercise without textual status
+        if context['full_score']:
+            points_type = "full"
+        elif context['passed']:
+            points_type = "passed"
+        elif context['submitted']:
+            points_type = "not_passed"
+    if not context['confirm_the_level'] and context.get('unconfirmed'):
+        points_type += " unconfirmed"
+    return points_type
```

Comment:
This returns string, but other functions expects a tuple.

Reply:
Changed into a list.